doc-content/enterprise-only/installation/installer-jws-proc.adoc

```diff
@@ -29,20 +29,20 @@ Ensure that you are logged in with a user that has write permission for Tomcat 9
 +
 [source]
```

Comment:
For all code snippets that use variables in them, update the `[source]` to be `[source,subs="attributes+"]`. Otherwise the variables don't render correctly in the output. And be sure to verify the output completely, especially since I can't see it, given the outdated previews in this JIRA.